gstreamer-sdp/src/s_d_p_message.rs

```diff
@@ -35,85 +35,84 @@ impl SDPMessage {
     pub fn new() -> SDPMessage {
         assert_initialized_main_thread!();
         unsafe {
-            let msg = glib_ffi::g_malloc(mem::size_of::<ffi::GstSDPMessage>());
-            ffi::gst_sdp_message_new(msg);
-            from_glib_full(msg)
+            let msg = mem::uninitialized();
+            from_glib_full(ffi::gst_sdp_message_new(&mut msg))
```

Comment:
This returns an `SDPResult`, not the actual message. Your message would be `msg`